tests/data/github/github_request_pull_request_1

```diff
@@ -1,387 +1,651 @@
 {
-    "_links": {
-        "comments": {
-            "href": "https://api.github.com/repos/zhquan_example/repo/issues/1/comments"
-        },
-        "commits": {
-            "href": "https://api.github.com/repos/zhquan_example/repo/pulls/1/commits"
-        },
-        "html": {
-            "href": "https://github.com/zhquan_example/repo/pull/1"
-        },
-        "issue": {
-            "href": "https://api.github.com/repos/zhquan_example/repo/issues/1"
-        },
-        "review_comment": {
-            "href": "https://api.github.com/repos/zhquan_example/repo/pulls/comments{/number}"
-        },
-        "review_comments": {
-            "href": "https://api.github.com/repos/zhquan_example/repo/pulls/1/comments"
-        },
-        "self": {
-            "href": "https://api.github.com/repos/zhquan_example/repo/pulls/1"
+    "url": "https://api.github.com/repos/chaoss/grimoirelab-tutorial/pulls/12",
```

Comment:
As I commented in this pull request's corresponding [issue](https://github.com/chaoss/grimoirelab-perceval/issues/516#issuecomment-487260094), since the zhquan_example repository does not have the "reviews_data" and the "reviews" keys, the test would fail always. 

I will undo this change (though I am still not sure what causes the tests to fail)